docs/mrt/README.md

```diff
@@ -14,9 +14,33 @@ git clone git@github.com:CortexFoundation/cvm-runtime.git
 
 2.2. Build the Shared Library
 
+This section gives an example of compiling CVM.
+
+1. Config the configuration for compilation
+
+   1. Check file `config.cmake` exists in your project root, or execute the following command:
+
+      ```bash
+      cp cmake/config.cmake .
+      ```
+
+   2. set the `ENABLE_CUDA` variable `ON` in `config.cmake` line 6.
+
+2. Compile with following command
+
+```bash
+make -j8 lib
```

Comment:
This introduction should move to the installation markdown file in the cvm module.

In fact, I think the installation or preparation of MRT is just the setup and
dependency of the python environment. MRT relies on the cvm python module which should be
elaborated in the cvm doc. Moreover, here may mainly acknowledge the python dependencies since
the setup of python already has been introduced in the cvm installation and you just need to mention
the python setup command to users and give a link of cvm python setup for more details.

Reply:
fixed